modules/virt-attaching-vm-to-ovn-secondary-nw-cli.adoc

```diff
@@ -28,20 +28,23 @@ spec:
       domain:
         devices:
           interfaces:
-          - name: secondary <1>
+          - name: secondary # <1>
             bridge: {}
         resources:
           requests:
             memory: 1024Mi
       networks:
-      - name: secondary  <2>
+      - name: secondary  # <2>
         multus:
-          networkName: <nad_name> <3>
+          networkName: <nad_name> # <3>
```

Comment:
we should want to clarify that nad name is a namespaced name. @sjhala-ccs is that consistent w/ the rest of the documentation ? 

@phoracek your thoughts ? You know how much I hate the entire notion of current project, and how much easier I think it is to **always** refer the namespace everywhere, but I'm unsure that would be consistent with the rest of the docuentation.

Reply:
The rest of our docs is the same - not mentioning anything about the namespace. I haven't heard field teams complain about that, so I think it's ok. It may be good to advertise the default/<nad> feature a little more (although it would be pretty much obsoleted with CUDN). Personally, I would not touch it in this PR.